k9mail/src/main/java/com/fsck/k9/helper/Utility.java

```diff
@@ -111,21 +111,22 @@ public static boolean requiredFieldValid(TextView view) {
 
 
     public static boolean requiredFieldValid(Editable s) {
+        return s != null && requiredFieldValid(s.toString());
```

Comment:
If there is no need to substitute those methods for testing purposes, then no. Otherwise, classes should ideally be transformed to non-static in one coherent change in a single commit